interop-keys/.gitignore

```diff
@@ -0,0 +1,2 @@
+payload.json
+result.json
```

Comment:
This will ignore any file anywhere with these names.  We probably should use a specific path or better yet the script could just create a temp dir to put these files in and delete on exit.  A temp dir would also solve the issue that currently the script is writing these files to whatever the current directory happens to be which isn't ideal.

Reply:
ugh ok i had assumed it worked like other ignore files. will remove that.